lib/tasks/custom_erd.rake

```diff
@@ -0,0 +1,13 @@
+namespace :erd do
+  task generate: :erd_with_timestamp
+
+  task :erd_with_timestamp do
+    unless ENV['title']
+      git_sha = %x{git rev-parse HEAD}[0...8]
+      timestamp = Time.now.utc.strftime("%Y-%m-%d")
```

Comment:
Yeah, I agree about excluding the timestamp or explicitly showing the timezone. I think the definite worst case long term is to not show the timezone but still use the user's local timezone rather than a timezone that's uniform across the project. (For this project, for example, always recording the current date in the Eastern timezone could make sense instead of UTC, but you'd want to still show ET in the title of the pdf to reduce confusion and since this a document for humans rather than computers.)

I think I'd lean toward still including the date (with timezone) just because that would indicate to a human much faster whether this reference document is relevant to the code they'll be working with, but that's really up to you and @latazzajones, since you're the primaries and will possibly be referring to this more than anyone else.